### PR TITLE
Finance Tables pattern

### PR DESCRIPTION
**Pull request by philipjohn**
This change provides a new "Finance Tables" pattern to address the need for the Balance Sheet and State of Activites sections of the report design (see #991).

The new pattern uses the Shiro Accordion block, along with heading and table blocks with specific styling targeting these blocks within report pages.

Screenshot 2024-03-20 at 20 17 55
Screenshot 2024-03-20 at 20 18 08
Screenshot 2024-03-20 at 20 18 40
Screenshot 2024-03-20 at 20 18 52

### CODE REVIEW

File: src/patterns/hero/frontend.scss [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+			font-family: Noto Sans;
+			font-size: #{$font-size-xxxlarge};
+			font-weight: bold;
+			color: white;
+			padding: 1em 0.5em 0.5em 0.1em;
+		}
+	}
+
+	.reports-hero-content {
+		padding: calc( ( 100vw - 719px ) * -0.3 + 50px ) 1.25rem 0 1.25rem;
```
Comment (goldenapples) [inline src/patterns/hero/frontend.scss:63]: This is a wild value for padding-top. What does it mean to be setting the top padding based on `vw`?

**Review by goldenapples — APPROVED** [review]
The tables look good to me. The only suggestion I have is that it would be good to set the column widths so that the figures column is aligned on each of the tables.

File: src/patterns/hero/frontend.scss (hunk 1 above)
Comment (philipjohn) [reply src/patterns/hero/frontend.scss:63]: This is the issue I'm having over here: https://github.com/wikimedia/wikimedia-wordpress-annual-report-plugin/pull/13#issuecomment-2009722730

**Review by mattwatsoncodes — CHANGES_REQUESTED** [review]
Spotted a few design issues that would be good to address:

Desktop: 
CleanShot 2024-03-22 at 21 48 43@2x

- There is no wide align option on the accordion, which is needed for this design
- The numbers column is not left aligned
- The spacing between the columns on each row should match
- There is a right and left padding on the accordion items, but in the design it goes edge to edge
- The titles are not centred

Desktop Design Reference:

CleanShot 2024-03-22 at 21 51 30@2x


Mobile:
CleanShot 2024-03-22 at 21 52 03@2x

- Feedback as above

Mobile Design Reference:
CleanShot 2024-03-22 at 21 52 50@2x

**Pull request merged**